src/fromager/sources.py

```diff
@@ -369,10 +369,10 @@ def prepare_source(
     version: Version,
 ) -> pathlib.Path:
     logger.info(f"{req.name}: preparing source for {req} from {source_filename}")
-    source_root_dir = overrides.find_and_invoke(
+    source_root_dir, _ = overrides.find_and_invoke(
```

Comment:
This will require changing the return value of all existing plugins to include the flag. We could have some more complicated code here to unpack the return value if it's a tuple or not if it's just a single string, and avoid breaking existing implementations.